From e953c2e991a815221c98131c2f9929ea602c7891 Mon Sep 17 00:00:00 2001
From: Alex Deymo <deymo@google.com>
Date: Wed, 24 Aug 2016 14:54:06 -0700
Subject: [PATCH 054/249] Remove intel edison from main manifest.

These repos are part of Brillo in the Brillo manifest and shouldn't be
included in the main Android manifest. These were added here:
b19edffa2d45114702c0613b45db39310a558fea

Change-Id: I0b67a97c7c7c21a48cb4f4b6f6690f6a417b03e0
---
 default.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/default.xml b/default.xml
index 8836746..649aa3c 100644
--- a/default.xml
+++ b/default.xml
@@ -331,7 +331,6 @@
   <project path="hardware/akm" name="platform/hardware/akm" remote="aosp" />
   <project path="hardware/broadcom/libbt" name="CyanogenMod/android_hardware_broadcom_libbt" groups="pdk" />
   <project path="hardware/broadcom/wlan" name="CyanogenMod/android_hardware_broadcom_wlan" groups="pdk,broadcom_wlan" />
-  <project path="hardware/bsp/intel" name="platform/hardware/bsp/intel" remote="aosp" />
   <project path="hardware/google/apf" name="platform/hardware/google/apf" groups="pdk" remote="aosp" />
   <project path="hardware/intel/audio_media" name="platform/hardware/intel/audio_media" groups="intel" remote="aosp" />
   <project path="hardware/intel/bootstub" name="platform/hardware/intel/bootstub" groups="intel" remote="aosp" />
-- 
2.17.1

